_data/other.yml

```diff
@@ -285,6 +285,12 @@ websites:
       software: Yes
       doc: http://doxfer.webmin.com/Webmin/Enhanced_Authentication
 
+    - name: Wikia
+      url: https://www.wikia.com/
```

Comment:
That URL redirects to `http://www.wikia.com/fandom`. Is that just me or is that really their URL?